src/gatsby-ssr.js

```diff
@@ -17,7 +17,7 @@ export const onRenderBody = ({ setHeadComponents }, pluginOptions) => {
   setHeadComponents(
     parsedDomains.map(({ domain, crossOrigin }) =>
       React.createElement('link', {
-        crossOrigin: crossOrigin,
+        crossOrigin: crossOrigin ? crossOrigin : undefined,
```

Comment:
Please add a comment reporting
```js
// Returning `undefined` in case of a falsy value prevents the following warning
// > Warning: Received `false` for a non-boolean attribute `crossOrigin`.
```
before the line you modified 😊 and following Conventional Commits, please squash the changes in a single commit with the `fix: Fix React warning when using crossOrigin false` message.

Thanks a lot, @mwskwong!!!